tests/test_assessment_engine.py

```diff
@@ -227,5 +228,10 @@ def test_assessments_csv(self):
             '/api/patient/assessment?format=csv&instrument_id={}'.format(instrument_id),
         )
         csv_string = download_response.data
```

Comment:
Try the [`get_data()` method](http://werkzeug.pocoo.org/docs/0.14/wrappers/#werkzeug.wrappers.BaseResponse.get_data) to get a unicode string, instead of a bytestring, then you shouldn't need the version-conditional code.
```
csv_string  = download_response.get_data(as_text=True)
```